packages/model/src/calendar.ts

```diff
@@ -1,17 +1,20 @@
 import { BasicCalendar, BasicCalendarResponse } from './calendar-common';
 import { FetchPaginationOptions, ListResponse } from './common';
-import { GroupDataObject, GroupResponse } from './group';
+import {
+  InterestGroupDataObject,
+  InterestGroupResponse,
+} from './interest-group';
 import { WorkingGroupDataObject, WorkingGroupResponse } from './working-group';
 
 export interface CalendarDataObject extends BasicCalendar {
-  groups?: Pick<GroupDataObject, 'id' | 'active'>[];
+  groups?: Pick<InterestGroupDataObject, 'id' | 'active'>[];
```

Comment:
Just saw this now, I think it would be nice to change it here as well
```suggestion
  interestGroups?: Pick<InterestGroupDataObject, 'id' | 'active'>[];
```